node/trieIterators/directStakedListProcessor.go

```diff
@@ -74,11 +74,13 @@ func (dslp *directStakedListProcessor) getAllStakedAccounts(validatorAccount sta
 			continue
 		}
 
+		staked := big.NewInt(0).Set(totalStakedCurrentAccount)
+		staked.Sub(staked, totalTopUpCurrentAccount)
 		val := &api.DirectStakedValue{
 			Address: dslp.publicKeyConverter.Encode(leafKey),
-			Staked:  totalStakedCurrentAccount.String(),
+			Staked:  staked.String(),
```

Comment:
The separation into Staked and Total is misleading, as topup is also staked.

Can the field "staked" be renamed into base stake?

Reply:
done